src/App/mermaidData/databaseSwitchboard/ProjectsMixin.js

```diff
@@ -247,6 +247,53 @@ const ProjectsMixin = (Base) =>
       return Promise.reject(this._notAuthenticatedAndReadyError)
     }
 
+    deleteProject = async function deleteProject(project, projectId) {
+      const hasCorrespondingRecordInTheApi = !!project._last_revision_num
```

Comment:
Ok, I get this now reading through this function. Maybe it can be named `hadCorrespondingProjectInTheApi` (because for this app 'record' has leaded meaning as a collect record)

Reply:
or 'has projectBeenPushedToApi`?

Reply:
ah right - the var was pasted from the collect record delete function - will update the name. I like ed `hadCorrespondingProjectInTheApi`